### PR TITLE
2023_04_11.md

### PR DESCRIPTION
**Pull request by markus2330**
## Basics

- [ ] Details of what you changed are in commit messages
      (first line should have `module: short statement` syntax)
- [ ] References to issues, e.g. `close #X`, are in the commit messages.
- [ ] The buildserver is happy.
- [ ] The PR is rebased with current master.

## Checklist

- [ ] I added unit tests for my code
- [ ] I fully described what my PR does in the documentation
      (not in the PR description)
- [ ] I fixed all affected documentation
- [ ] I fixed all affected decisions
- [ ] I added code comments, logging, and assertions as appropriate
- [ ] I mentioned [every code or binary](/.reuse/dep5) not directly written or done by me in [reuse syntax](https://reuse.software/)

## Review

- [ ] Documentation is conforming to [our Documentation Guidelines](/doc/documentation.md)
- [ ] Examples are well chosen and understandable
- [ ] Code is conforming to our Coding Guidelines
- [ ] Code is consistent to [our Design Decisions](/doc/decisions)

### CODE REVIEW

File: doc/meetings/2023_04_07_db.md [hunk 1]
```diff
@@ -0,0 +1,77 @@
+# Meeting 2023-04-07 DB
+
+_Protocolled by: Nursultan_
+
+## Attendees
+
+- Nursultan
+- Moritz
```
Comment (badnames) [inline doc/meetings/2023_04_07_db.md:8]:
```suggestion
```
I could not attend this meeting unfortunately.

Comment (markus2330) [reply doc/meetings/2023_04_07_db.md:8]: @badnames it will be today, it was moved.

Comment (markus2330) [reply doc/meetings/2023_04_07_db.md:8]: In one hour :wink:

**Review by aidnurs — APPROVED** [review]

**Comment by markus2330** [comment]
@NurramoX thx!

**Pull request merged**